### DISCUSSION TITLE
BigSur- Monterey- OC-0.8.2 Release- Infinix

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#2] [Q&A] [answered]
- Updated to OpenCore 8.2.
- Reworked EFI
- Fixed Power Management for i3 model
- Fixed Thermals
- Fixed Core stuff in EFI
- Fixed grammer in EFI
- Removed unnecessary Kexts, Drivers, Tools in EFI
- Changed SMBios from `MacBookPro 16,2` to `MacBookAir9,1` for better Co-ordination with macOS and also to increase performance and power Management for i3,i5,i7 Models.
-  Everything is working with BigSur.
- Only Bluetooth is dead, due to Infinix bluetooth firmware doesn't support Monterey due to its in alpha (the issue is from development, not in this EFI).
- Fixed Trackapd in Monterey(Trackpad is now functional as like BigSur)
- Increased Stability.
- Fixed USBs.
- Reworked ACPI.
- Increase trackpad and keyboard response. So, the response is quick.
- Add SMBios. your wish. But use `MacBookAir9,1` smbios.
- ### Note- Don't change anything, because every option is set for a reason.
- ### Don't `Clean-snapshot` in proper tree, you will lose all the config settings and its automatically sets into OpenCore default settings in `config.plist`. Don't do it unless you know what you are doing. So, use Snapshot in proper tree .

<hr /><em>This discussion was created from the release <a href='https://github.com/devboloji/Infinix-Hackintosh-Guide-Opencore_INBook_X1_XL11_i3-/releases/tag/OC-8.2'>BigSur- Monterey- OC-0.8.2 Release- Infinix</a>.</em>

**Comment by extasea** [comment, marked as answer]
Hi Mate, first of all thanks for the EFI. It help me a lot with the hackintosh

I got problem with the booting, while the apple logos appear the system restart over and over again.
It takes some time to boot normally to the bigsur

I use BigSur 11.6.8, the latest one

Kindly help me with this problem
Thank you

**Reply by devboloji** [reply]
@extasea can you explain clearly...again?

**Reply by babibudo** [reply]
does the big sur version matter? haven't tried 11.6.8 but the ollarilla raw file of 11.6.7 works fine on me.

**Reply by devboloji** [reply]
@babibudo The BigSur version doesn't matter. I tested, running BigSur 11.6.8 from 11.6.5 via system update. No errors and issues faced and running Great. Also updated to Monterey from BigSur and updated Monterey 12.4 to 12.5. Same no errors faced. So, everyone can use or update to latest versions of macOS11and 12. Don't ask for ventura for now, because of its in beta and efi don't work. It's a waste of time for builting it for now.
I have tested the EFI in many ways to give you rich experience by spending my time. So donating me is worth.

**Reply by devboloji** [reply]
@extasea hello, Firstly remember even the Latest M1 MacBook takes atleast > 30sec for booting, and Intel MacBook takes more-than 1min for booting. So our Infinix device boots with in < 30sec. 
Secondly,  When using hackintosh and booting the device, actually you see the text moving for some time. Then, the apple logo shows loading.(this is because of the OS is checking every file by approving files in the EFI or if there is any mistake in the EFI, it's easy to find what the issue is by the help of those text).
But,  In place of the text, we are tricking the text 
 to show the apple logo. So you see booting twice. But in reality, the device boots once.
I hope you found your answer

**Reply by extasea** [reply]
Thanks for the response @devboloji 
The problem solved, it was the second SSD slot.
After the second SSD removed, the machine can boot normally.
I don't know why, maybe my device defect or something

**Reply by devboloji** [reply]
@babibudo  @extasea great. if my EFI is helpful, donation will be helpful for the future releases to this device. [Donate here](https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/blob/main/README.md#any-one-can-donate-me-or-buy-me-a-coffee-)

**Reply by ahmadgunn** [reply]
Hi Dev, first of all thanks for the EFI. 

I got problem with the sleep, while the laptop sleep it goes restart.

I use BigSur 11.6.7.

Kindly help me with this problem
Thank you

This is the report :
panic(cpu 3 caller 0xffffff800afc3356): Kernel trap at 0xffffff80103a5d45, type 14=page fault, registers:
CR0: 0x000000008001003b, CR2: 0x00000000000000c8, CR3: 0x0000000012675000, CR4: 0x00000000003626e0
RAX: 0xffffffb07d1bbd90, RBX: 0x0000000000000000, RCX: 0xffffff801039aeda, RDX: 0xffffffa02fe32a60
RSP: 0xffffffb07d1bbd70, RBP: 0xffffffb07d1bbd80, RSI: 0xffffffb07d1bbd90, RDI: 0x0000000000000000
R8:  0x0000000000000093, R9:  0x0000000000000010, R10: 0x0000000000000000, R11: 0x0000000000000008
R12: 0xffffff9352ff95c0, R13: 0xffffff801039f468, R14: 0xffffffb07d1bbd90, R15: 0x0000000000000000
RFL: 0x0000000000010286, RIP: 0xffffff80103a5d45, CS:  0x0000000000000008, SS:  0x0000000000000010
Fault CR2: 0x00000000000000c8, Error code: 0x0000000000000000, Fault CPU: 0x3, PL: 0, VF: 1

Backtrace (CPU 3), Frame : Return Address
0xffffffb07d1bb790 : 0xffffff800ae8b26d 
0xffffffb07d1bb7e0 : 0xffffff800afd2993 
0xffffffb07d1bb820 : 0xffffff800afc2f8a 
0xffffffb07d1bb870 : 0xffffff800ae2fa2f 
0xffffffb07d1bb890 : 0xffffff800ae8aa8d 
0xffffffb07d1bb9b0 : 0xffffff800ae8ad83 
0xffffffb07d1bba20 : 0xffffff800b69c5fa 
0xffffffb07d1bba90 : 0xffffff800afc3356 
0xffffffb07d1bbc10 : 0xffffff800afc303d 
0xffffffb07d1bbc60 : 0xffffff800ae2fa2f 
0xffffffb07d1bbc80 : 0xffffff80103a5d45 
0xffffffb07d1bbd80 : 0xffffff801039afcf 
0xffffffb07d1bbdb0 : 0xffffff801039f4b0 
0xffffffb07d1bbde0 : 0xffffff800b5f31d5 
0xffffffb07d1bbe50 : 0xffffff800b5f30d6 
0xffffffb07d1bbe80 : 0xffffff800aed1985 
0xffffffb07d1bbef0 : 0xffffff800aed28f4 
0xffffffb07d1bbfa0 : 0xffffff800ae2f13e 
      Kernel Extensions in backtrace:
         science.firewolf.rtsx(0.9.6)[7182D818-35D8-3B1C-A38F-D1D7920DC055]@0xffffff801038f000->0xffffff8010426fff
            dependency: com.apple.iokit.IOPCIFamily(2.9)[4169693C-CDDE-3CA7-800A-F665762B5B4C]@0xffffff800d96c000->0xffffff800d994fff
            dependency: com.apple.iokit.IOStorageFamily(2.1)[12E52DC2-DD26-3C9A-94CD-07C7732363A7]@0xffffff800da6f000->0xffffff800da80fff
            dependency: com.apple.iokit.IOUSBHostFamily(1.2)[748FD795-BA14-3BFA-ACF4-A47AE745021F]@0xffffff800dc8a000->0xffffff800dd17fff

Process name corresponding to current thread: kernel_task
Boot args: -v alcid=99 watchdog=0 igfxonln=1 agdpmod=vit9696 -vi2c-force-polling -igfxblr -noDC9 igfxfw=2 -igfxdbeo -wegnoegpu 

Mac OS version:
20G630

Kernel version:
Darwin Kernel Version 20.6.0: Tue Apr 19 21:04:45 PDT 2022; root:xnu-7195.141.29~1/RELEASE_X86_64
Kernel UUID: 76569598-07D6-3661-974C-6C024B198E5B
KernelCache slide: 0x000000000ac00000
KernelCache base:  0xffffff800ae00000
Kernel slide:      0x000000000ac10000
Kernel text base:  0xffffff800ae10000
__HIB  text base: 0xffffff800ad00000
System model name: MacBookAir9,1 (Mac-0CFF9C7C2B63DF8D)
System shutdown begun: NO
Panic diags file available: YES (0x0)
Hibernation exit count: 0

System uptime in nanoseconds: 1427518988564
Last Sleep:           absolute           base_tsc          base_nano
  Uptime  : 0x0000014c5ec32389
  Sleep   : 0x0000014c2794da90 0x0000000334013138 0x0000000000000000
  Wake    : 0x0000014c29616484 0x0000000049a40493 0x0000014c28522864

**Reply by devboloji** [reply]
@ahmadgunn have u disable those required settings in BIOS?

**Reply by devboloji** [reply]
@ahmadgunn  and what version of my efi you used?

**Reply by ahmadgunn** [reply]
@devboloji yes i do anything those required setting in bios, the latest one.

**Reply by devboloji** [reply]
@ahmadgunn did you use any hibernation apps?
And explain me what have you before the issue raised

**Reply by ahmadgunn** [reply]
No i dont have any hibernation apps.
The issue is there after while i install the hackintosh

**Reply by devboloji** [reply]
reinstall the os

**Reply by devboloji** [reply]
@ahmadgunn @babibudo @extasea 
Join the Telegram group for any discussion of infinix, [Group](https://t.me/infinix_inbook_Updates)
